src/Angor/UniSatWallet/Exceptions/NoUniSatWalletException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UniSatWallet.Exceptions
```

Comment:
If we have a project for each wallet that could become a very big solution, perhaps a project for wallet connectors with a connector and models file per wallet?